docs/ops/config.rst

```diff
@@ -195,9 +195,9 @@ in other Django projects.
 
     The account password to authenticate as DJANGO_REMOTE_SETTINGS_USERNAME.
 
-.. envvar:: DJANGO_REMOTE_SETTINGS_COLLECTION_ID
+.. envvar:: REMOTE_SETTINGS_CAPABILITIES_COLLECTION_ID
```

Comment:
The `DJANGO_` prefix is still required. Almost of the our settings that use `django-configurations` use that prefix by default.